test/Microsoft.Data.Sqlite.Tests/SqliteConnectionStringBuilderTest.cs

```diff
@@ -174,13 +175,19 @@ public void ShouldSerialize_returns_false_when_not_exists()
             Assert.False(new SqliteConnectionStringBuilder().ShouldSerialize("Invalid"));
         }
 
-        [Fact]
+        [ConditionalFact]
+        [FrameworkSkipCondition(
+            RuntimeFrameworks.Mono,
```

Comment:
As a future improvement to testing, it would be better to skip this only on Mono 4.0.2 to 4.0.4, not all of mono.